huaweicloud/services/fgs/resource_huaweicloud_fgs_dependency_version.go

```diff
@@ -38,100 +38,126 @@ func ResourceDependencyVersion() *schema.Resource {
 			},
 			"name": {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
-				Description: "The name of the custom dependency package to which the version belongs.",
+				Description: `The name of the custom dependency package to which the version belongs.`,
 			},
 			"link": {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
-				Description: "The OBS bucket path where the dependency package is located.",
+				Description: `The OBS bucket path where the dependency package is located.`,
 			},
 			"description": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Computed:    true,
 				ForceNew:    true,
-				Description: "The description of the custom dependency version.",
+				Description: `The description of the custom dependency version.`,
 			},
 			"version": {
 				Type:        schema.TypeInt,
 				Computed:    true,
-				Description: "The dependency package version.",
+				Description: `The dependency package version.`,
 			},
 			"version_id": {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "The ID of the dependency package version.",
+				Description: `The ID of the dependency package version.`,
 			},
 			"owner": {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "The dependency owner, public indicates a public dependency.",
+				Description: `The dependency owner, public indicates a public dependency.`,
 			},
 			"etag": {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "The unique ID of the dependency.",
+				Description: `The unique ID of the dependency.`,
 			},
 			"size": {
 				Type:        schema.TypeInt,
 				Computed:    true,
-				Description: "The dependency size, in bytes.",
+				Description: `The dependency size, in bytes.`,
 			},
 			"dependency_id": {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "The ID of the dependency package corresponding to the version.",
+				Description: `The ID of the dependency package corresponding to the version.`,
 			},
 		},
 	}
 }
 
-func buildDependencyVersionOpts(d *schema.ResourceData) dependencies.DependVersionOpts {
+func buildCreateDependencyVersionBodyParams(d *schema.ResourceData) map[string]interface{} {
 	// Since the ZIP file upload is limited in size and requires encoding, only the OBS type is supported.
 	// The ZIP file uploading can also be achieved by uploading OBS objects and is more secure.
-	return dependencies.DependVersionOpts{
-		Name:        d.Get("name").(string),
-		Runtime:     d.Get("runtime").(string),
-		Description: utils.String(d.Get("description").(string)),
-		Type:        "obs",
-		Link:        d.Get("link").(string),
+	return map[string]interface{}{
+		"name":        d.Get("name").(string),
+		"runtime":     d.Get("runtime").(string),
+		"description": utils.ValueIgnoreEmpty(d.Get("description").(string)),
```

Comment:
This line of code is different from the original logic, please keep it consistent.

Reply:
done.